java-license.txt

```diff
@@ -0,0 +1,14 @@
+/* Copyright 2016 Google Inc
```

Comment:
This file isn't the license for java; maybe call the filename "license-header-javadoc.txt"